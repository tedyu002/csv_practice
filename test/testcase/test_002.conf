CSV_INPUT=test_002.conf.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
HEADERS=Year CHAR(6),Brand VARCHAR(64),Type VARCHAR(12),Price INTEGER,Considered BOOL
CSV_FORMULA=Print Considered[3] + Price[4] * 2
SORT_HEADERS=Considered,Price
SORT_ORDER=ASC
