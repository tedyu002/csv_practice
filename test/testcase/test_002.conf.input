1997,Ford Escape V1,E 350,800000,TRUE
1991,Mercedes-Benz Cloud First Time,S 320,2800000,TRUE
2003,Mercedes-Benz,C 220,1800000,TRUE
2016,Volvo Openfind I love you,S 60,1750000,FALSE
1991,Audi,A 8,1850000,TRUE
2003,Mazda SOA,CX-5,1150000,FALSE
