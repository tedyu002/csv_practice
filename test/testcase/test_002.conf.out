FALSE,1150000,"  2003","Mazda SOA","CX-5"
FALSE,1750000,"  2016","Volvo Openfind I love you","S 60"
TRUE,800000,"  1997","Ford Escape V1","E 350"
TRUE,1800000,"  2003","Mercedes-Benz","C 220"
TRUE,1850000,"  1991","Audi","A 8"
TRUE,2800000,"  1991","Mercedes-Benz Cloud First Time","S 320"
